### PR TITLE
Fix https://github.com/cuberite/cuberite/issues/3339

### PR DESCRIPTION
**Pull request by mohe2015**

### CODE REVIEW

File: web_ranks.lua [hunk 1]
```diff
@@ -455,6 +455,9 @@ local function ShowSaveVisualsPage(a_Request)
 		return HTMLError("Invalid request received, missing values.")
 	end
 	
+	if (g_ColorCodeDef[MsgNameColorCode] == nil) then
```
Comment (NiLSPACE) [inline web_ranks.lua]: Personally I prefer
```lua
if (not g_ColorCodeDef[MsgNameColorCode]) then
...
end
```

**Comment by mohe2015** [comment]
Changed that.

**Comment by NiLSPACE** [comment]
Thanks, could you squash all the commits together?

**Comment by mohe2015** [comment]
How can I do that?

**Comment by mohe2015** [comment]
Figured it out myself.

**Comment by NiLSPACE** [comment]
Great, thank you :)

**Comment by madmaxoft** [comment]
It would be great if next time you put the actual problem description into the PR name and commit message, instead of a link to another issue. It would make displaying history much better to understand.

**Comment by sphinxc0re** [comment]
PR template? 😄

**Comment by mohe2015** [comment]
@madmaxoft Ok, I'll to that.

**Pull request merged**